npm/xpath/cypress/support/e2e.js

```diff
@@ -0,0 +1,16 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+import '../../src'
```

Comment:
The video recording should be deleted. & need to update the root .gitignore to ignore this so they aren't checked in to github on accident